packages/commonwealth/client/scripts/views/components/component_kit/cw_icons/cw_icons.tsx

```diff
@@ -3216,6 +3216,37 @@ export const CWTwitterNew = (props: IconProps) => {
   );
 };
 
+export const CWTwitterX = (props: IconProps) => {
```

Comment:
we have this already, maybe rename it? https://github.com/hicommonwealth/commonwealth/blob/ac8eea2c5ae7ae3e8ec46483150b5fe58fc6cfd6/packages/commonwealth/client/scripts/views/components/component_kit/cw_icons/cw_icon_lookup.ts#L255